feat(ec): new data directory location and configuration (#2723)
replicatedhq/replicated-docs: docs/enterprise/installing-embedded.mdx

### Limitation

It is not possible to change the port for the Admin Console during a restore with Embedded Cluster. For more information, see [Disaster Recovery for Embedded Cluster (Alpha)](/vendor/embedded-disaster-recovery).

> ## Change the Default Data Directory {#data-dir}

Reviewer: ^ Added "Change the Default Data Directory" section in the Embedded Cluster online install topic next just below the existing section of the changing the default kotsadm/LAM ports

The default location of the data directory for Embedded Cluster is `/var/lib/embedded-cluster`.

This directory can be modified by the user by passing the `--data-dir` flag to the `install` command.

For example:

```bash
sudo ./APP_SLUG install --data-dir /data/embedded-cluster --license license.yaml --airgap-bundle APP_SLUG.airgap
```
Where `APP_SLUG` is the unique application slug.

> ### Limtiations

Reviewer: ^ Added limitations related to changing the default dir, including that you have to use the same dir during a restore

* The data directory for Embedded Cluster cannot be changed after the cluster is installed.

* If you use the `--data-dir` flag to change the data directory during installation, then you must use the same location when restoring in a disaster recovery scenario. For more information about disaster recovery with Embedded Cluster, see [Disaster Recovery for Embedded Cluster](/vendor/embedded-disaster-recovery).

> * Replicated does not support using symlinked directories for the Embedded Cluster data diretory. Instead, Replicated recommends bind mounts, which provide greater consistency across different applications, preserve the original directory permissions and ownership, and have less overhead compared to symlinks. For more information, see [Relocating data directories with bind mounts](https://community.replicated.com/t/relocating-data-directories-with-bind-mounts/1435) in Replicated Community.

Reviewer 2: We don't need to recommend bind mounts anymore because this flag lets you choose an alternate directory without needing to bind mount. So we should still say "no symlinks" (in fact, we're going to preflight and ensure symlinks aren't used), but we should recommend `--data-dir` over bind mounts now.
```suggestion
* Replicated does not support using symlinked directories for the Embedded Cluster data directory. Use the `--data-dir` flag instead of symlinking `/var/lib/embedded-cluster`.
```
